src/Utopia/Messaging/Adapters/SMS/Termii.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+// Reference Material
+// https://developers.termii.com/messaging-api
+class Termii extends SMSAdapter
+{
+    const SMSType = 'plain';
+
+    const SMSChannel = 'generic';
+
+    public function __construct(
+        private string $apiKey
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return 'Termii';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 100;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
+    protected function process(SMS $message): string
+    {
+        return $this->request(
+            method: 'POST',
+            url: 'https://api.ng.termii.com/api/sms/send',
+            headers: [
+                'Content-Type: application/json',
+            ],
+            body: \json_encode(
+                $this->getRequestBody(
+                    to: $message->getTo(),
+                    text: $message->getContent(),
+                    from: $message->getFrom()
+                )
+            ),
+        );
+    }
+
+    /**
+     * Get the request body
+     *
+     * @param  array  $to  Phone number
+     * @param  string  $text  Message to send
+     * @param  string|null  $from Origin of the message
+     */
+    private function getRequestBody(array $to, string $text, string $from = null): array
+    {
+        $from = $from ?? '';
+
+        // removing + from numbers if exists
+        $to = \array_map(
+            fn ($to) => \ltrim($to, '+'),
+            $to
+        );
+
+        if (count($to) == 1) {
+            $to = $to[0];
+        }
```

Comment:
Do they not accept an array with 1 element?

Reply:
not sure about that, based on the documentation they just mentioned that  to send to multiple numbers we need to pass as array

Reply:
It would be good to test and see so that you can clean up this code.

Reply:
Hi, I am unable to test the service anymore as my account balance is low. Termii does not provide free testing and since I am testing from India cost is high.

`Exception: {"message":"Insufficient balance"}`